code/modules/antagonists/superheroes/buzzon/buzzon_equipment.dm

```diff
@@ -0,0 +1,294 @@
+//Buzzon, controls all the robots and honey.
+
+/obj/item/clothing/gloves/tackler/combat/insulated/buzzon
+	name = "BuzzOn gloves"
+	desc = "Legendary gloves on the greatest bee superhero - BuzzOn. You can see small rocket engines installed on them."
+	icon_state = "buzz"
+	worn_icon_state = "buzz"
+	inhand_icon_state = "ygloves"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
+
+	tackle_stam_cost = 5 //Very light and fast tackles. However, if you fly into a wall or an enemy, well, you're fucked.
+	base_knockdown = 0.2 SECONDS
+	tackle_range = 5
+	tackle_speed = 2
+	min_distance = 2
+	skill_mod = -5
+
+/obj/item/clothing/gloves/tackler/combat/insulated/buzzon/cryo //Cosmetics for "Operation: Cryostung" kit
+	icon_state = "buzz_cryo"
+	worn_icon_state = "buzz_cryo"
+	inhand_icon_state = "bluegloves"
+
+/obj/item/clothing/shoes/sneakers/buzzon
+	name = "BuzzOn shoes"
+	desc = "Shoes of a bee that became human, rumors say that they buzz when you walk in them."
+	greyscale_colors = "#000000#f6c61a"
+
+	armor = list(MELEE = 60, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 55, BIO = 100, RAD = 70, FIRE = 100, ACID = 100, WOUND = 25)
```

Comment:
does this give increased armor to their wearer's legs, like combat boots do?

Reply:
Yes

Reply:
To their feet, not to their legs